internal/commands/server/create.go

```diff
@@ -6,6 +6,7 @@ import (
 	"os"
 	"strings"
 
+	"github.com/UpCloudLtd/upcloud-cli/internal/commands/ipaddress"
```

Comment:
put internal lib all together separated by a line with the external libs

Reply:
changed my IDE config to use goimports with `-local` flag which seems to do the trick.